internal/pkg/cli/local_run.go

```diff
@@ -462,6 +463,9 @@ func (o *localRunOpts) cleanUpContainers(ctx context.Context, containerURIs map[
 	}
 
 	if len(errs) > 0 {
+		sort.Slice(errs, func(i, j int) bool {
```

Comment:
It seems like the order of execution is (synchronously):
```
# cleanup container1
docker stop container1
docker rm container1

# cleanup container2
docker stop container2
docker rm container2

# cleanup pause
docker stop pause
docker rm pause
```

I feel like we want to maintain the order of the errors, such that errors emitted for the same containers are next to each other, instead of for example `stopContainer1Err, rmContainer2Err, rmContainer1Err`. Sorting the errors alphabetically would scramble that order. 

I _think_ the reason why the error messages come in random order was because `containerURIs` was unsorted. Maybe we can try sort that instead of the errors!